### DISCUSSION TITLE
JuiceFS在csi模式下使用时，会创建多个JuiceFS 客户端吗，想了解下目前的实现方法以及优缺点

### DISCUSSION THREAD

**Discussion by ActivePeter** [juicedata/juicefs#5054] [General]
一个pv&pvc是对应一个juicefs客户端；还是说，一个物理节点上共用一个juiceFS客户端

**Comment by zxh326** [comment]
这里有介绍默认的使用方式及其其他方式的优缺点。

https://juicefs.com/docs/zh/csi/introduction/
